src/page/template/modal/group-creation.htm

```diff
@@ -1,15 +1,15 @@
 <div id="group-creation-modal" data-bind="with: $root.groupCreation" class="group-creation__modal">
-  <modal params="isShown: isShown, onClosed: afterHideModal">
+  <modal params="isShown: isShown, onClosed: afterHideModal, ariaLabelby: 'group-creation-label', lock: true">
     <div class="modal__header">
       <!-- ko if: stateIsParticipants() -->
-        <div class="modal__header__title"  data-bind="text: participantsHeaderText" data-uie-name="status-people-selected"></div>
-        <div class="group-creation__action enabled accent-text" data-bind="click: clickOnCreate, text: participantsActionText" data-uie-name="do-create-group"></div>
-        <arrow-left-icon class="modal__header__button" data-bind="click: clickOnBack" data-uie-name="go-back"></arrow-left-icon>
+        <h2 id="group-creation-label" class="modal__header__title" data-bind="text: participantsHeaderText" data-uie-name="status-people-selected"></h2>
+        <button class="group-creation__action enabled accent-text button-reset-default" type="button" data-bind="click: clickOnCreate, text: participantsActionText, attr: { 'aria-label': participantsActionText }" data-uie-name="do-create-group"></button>
```

Comment:
pure curiosity question: you sometime add the `button-reset-default` class in the css files and here in the html, what is your usual decision criteria for this?

Reply:
Valid point let's keep it consistent moving this one into css